frontend/src/components/experimentRuns/ExperimentRunsListContent.js

```diff
@@ -70,8 +70,11 @@ const getTableColumns = (runTypes, instruments, launchesById) => [
     dataIndex: "run_processing_launch_time",
     sorter: true,
     render: (_, experimentRun) => (
-      <div style={{minWidth: "12rem"}}>
-        <ExperimentRunLaunchCard experimentRun={experimentRun} experimentRunLaunch={launchesById[experimentRun.id]}/>
+      <div style={{ minWidth: "12rem" }}>
+        {
+          experimentRun.platform !== "AXIOM_ARRAY" &&
```

Comment:
ugh ... 
what about testing for runTypes.itemsByID[experimentRun.run_type]?.needs_run_processing ?
This is much too specific...